<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>518</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>A7kili</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="pages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page1">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>416</width>
        <height>454</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="date">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>date</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <layout class="QHBoxLayout" name="Users_HL">
         <item>
          <widget class="QTextEdit" name="discussion">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>192</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="Notification_HL">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="imageutilisateur">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../client2-build-desktop/User3.png</normaloff>../client2-build-desktop/User3.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>75</width>
             <height>61</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nomutilisateur">
           <property name="text">
            <string>nom</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <layout class="QHBoxLayout" name="Dest_HL">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="desLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Destinataire:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Destinataire">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0" colspan="3">
        <layout class="QHBoxLayout" name="SendMessage_HL">
         <item>
          <widget class="QLabel" name="msgLabel">
           <property name="text">
            <string>Message   :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="msg">
           <property name="minimumSize">
            <size>
             <width>128</width>
             <height>51</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="envoyer">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>51</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>85</width>
             <height>51</height>
            </size>
           </property>
           <property name="text">
            <string>Envoyer</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="Lnew_messages">
             <property name="font">
              <font>
               <pointsize>7</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Nouveaux Messages</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="messagenonlu">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>73</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>73</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>335</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>Date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Heure</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Destination</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Message</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="supprimer">
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>118</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="copier">
           <property name="text">
            <string>Transférer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>479</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionChat"/>
    <addaction name="separator"/>
    <addaction name="actionChanger_mot_de_passe"/>
    <addaction name="separator"/>
    <addaction name="actionDeconnection"/>
   </widget>
   <widget class="QMenu" name="menuMessages">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Messages</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionMessages_recus"/>
    <addaction name="separator"/>
    <addaction name="actionMessages_envoy_s"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuMessages"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMessages_recus">
   <property name="text">
    <string>Messages recus</string>
   </property>
  </action>
  <action name="actionMessages_envoy_s">
   <property name="text">
    <string>Messages envoyés</string>
   </property>
  </action>
  <action name="actionChat">
   <property name="icon">
    <iconset>
     <normaloff>../client2-build-desktop/110_F_29508441_ZnTtslBUjHe44nuBtMBMqoM7aFJnbrS0.jpg</normaloff>../client2-build-desktop/110_F_29508441_ZnTtslBUjHe44nuBtMBMqoM7aFJnbrS0.jpg</iconset>
   </property>
   <property name="text">
    <string>Chat</string>
   </property>
  </action>
  <action name="actionChanger_mot_de_passe">
   <property name="icon">
    <iconset>
     <normaloff>../client2-build-desktop/password.png</normaloff>../client2-build-desktop/password.png</iconset>
   </property>
   <property name="text">
    <string>Changer mot de passe</string>
   </property>
  </action>
  <action name="actionDeconnection">
   <property name="icon">
    <iconset>
     <normaloff>../client2-build-desktop/343507.png</normaloff>../client2-build-desktop/343507.png</iconset>
   </property>
   <property name="text">
    <string>Deconnection</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
